packages/neuron-wallet/src/utils/ckb-rpc.ts

```diff
@@ -2,7 +2,7 @@ import type { ParamsFormatter } from '@ckb-lumos/rpc'
 import type { Block } from '@ckb-lumos/base'
 import { HexString } from '@ckb-lumos/base'
 import { CKBRPC } from '@ckb-lumos/rpc'
-import { Method } from '@ckb-lumos/rpc/lib/method'
+import { Method as Method_ } from '@ckb-lumos/rpc/lib/method'
```

Comment:
Does `Method_` conform to a specific naming convention?

Reply:
i've renamed to `SdkRpcMethod` for more understandable